paasta_tools/kubernetes_tools.py

```diff
@@ -1570,6 +1570,11 @@ def get_pod_template_spec(
         if prometheus_shard:
             labels["paasta.yelp.com/prometheus_shard"] = prometheus_shard
 
+        # Allow Kubernetes Services to easily find
+        # pods belonging to a certain smartstack namespace
+        for registration in self.get_registrations():
+            labels[f"paasta.yelp.com/service/{registration}"] = "true" # type: ignore
```

Comment:
I don't know whether we've thought about a hierarchy like this, but I'd be more inclined to continue using the `registrations` nomenclature (so `paasta.yelp.com/registrations/{registration}`).

@keymone can you weigh in here - I know you've thought about this a lot in the past

Reply:
generally speaking we should keep label names as flat as possible, but in this case i think going one level deep is warranted. i do agree that `registrations` is better than `service`.